CI/samples.ci/client/petstore/java/test-manual/jersey1/ApiClientTest.java

```diff
@@ -102,14 +102,9 @@ public void testSelectHeaderContentType() {
     public void testGetAuthentications() {
         Map<String, Authentication> auths = apiClient.getAuthentications();
 
-        Authentication auth = auths.get("api_key");
-        assertNotNull(auth);
-        assertTrue(auth instanceof ApiKeyAuth);
-        ApiKeyAuth apiKeyAuth = (ApiKeyAuth) auth;
-        assertEquals("header", apiKeyAuth.getLocation());
-        assertEquals("api_key", apiKeyAuth.getParamName());
-
-        auth = auths.get("petstore_auth");
+        assertNull(auths.get("api_key"));
```

Comment:
why does auths.get("api_key") return null?

I remember some endpoints in the petstore spec are defined to use "api_key" for authentication

Reply:
As per https://github.com/OpenAPITools/openapi-generator/pull/2861#issuecomment-493825852 we now have hasTokenAuthMethods in DefaultGenerator which controls whether an ApiKeyAuth is registered in the authentications map.  Sounds like you're saying that is not working - any idea why not?